mmdet/datasets/custom.py

```diff
@@ -85,7 +85,11 @@ def __init__(self,
                                               self.proposal_file)
         # load annotations (and proposals)
         self.data_infos = self.load_annotations(self.ann_file)
-
+        
```

Comment:
You should not modify the custom.py code, you can think of a more reasonable way